config.py

```diff
@@ -241,18 +252,28 @@ class Config:
             "build_timeout": 300,
         },
         "ec2-m5-4xlarge-us-east-2": {
-            "info": "Supported benchmark langs: R",
+            "info": "Supported benchmark langs: Python, R, JavaScript",
             "default_filters": {
                 "arrow-commit": {
                     "langs": {
-                        "R": {"names": ["tpch"]},
```

Comment:
Note this removes tpch from the m5 machine due to time. I know tpch was the purpose of creating the M5 in the first place but we already run SF 10k on `arm64-m6g-linux-compute`.